### PR TITLE
I've created the Modal for sign up and Removed the h1

### PR DESCRIPTION
**Pull request by rgossomTechtonic**
Sign up Modal for landing page.

### CODE REVIEW

File: landing-page.html [hunk 1]
```diff
@@ -14,10 +14,56 @@
...
   <div class="login-fields">
 
+    <!-- Button trigger modal -->
+    <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#exampleModal">
+      Sign up
+    </button>
+
+    <!-- Modal -->
+    <div class="modal fade" id="ModalSignUp" tabindex="-1" role="dialog" aria-labelledby="ModalLabelSingUp" aria-hidden="true">
+  <div class="modal-dialog" role="document">
```
Comment (umorenovanderlaanTechtonic) [inline landing-page.html]: Let's make sure to use nesting indentation where appropriate!

Comment (rgossomTechtonic) [reply landing-page.html]: Ok, I'll try to fix the nesting issue and send a new pull request.

File: landing-page.html [hunk 2]
```diff
@@ -14,10 +14,56 @@
 </nav>
 
 <body>
-  <h1></h1>
+  <!-- Sign-up Modal -->
   <div class="login-fields">
 
+    <!-- Button trigger modal -->
+    <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#exampleModal">
```
Comment (jhernandezinzunzaTechtonic) [inline landing-page.html]: the data-target property must be set to the id of the modal to open correctly, please make sure that the target and modal id names match.

File: landing-page.html [hunk 3]
```diff
@@ -14,10 +14,56 @@
...
+      Sign up
+    </button>
+
+    <!-- Modal -->
+    <div class="modal fade" id="ModalSignUp" tabindex="-1" role="dialog" aria-labelledby="ModalLabelSingUp" aria-hidden="true">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
```
Comment (jhernandezinzunzaTechtonic) [inline landing-page.html]: Lets try and fix the nesting issues, let me know if you need help!

File: landing-page.html [hunk 4]
```diff
@@ -24,13 +24,14 @@
 
     <!-- Modal -->
     <div class="modal fade" id="ModalSignUp" tabindex="-1" role="dialog" aria-labelledby="ModalLabelSingUp" aria-hidden="true">
```
Comment (jhernandezinzunzaTechtonic) [inline landing-page.html]: Indentations look good!

Nice job.

**Pull request merged**